Tasks/Src/RTOS_BPS.c

```diff
@@ -19,6 +19,12 @@ void RTOS_BPS_MutexPost(void){
 void RTOS_BPS_TaskCreate(void){
 
 }
-void RTOS_BPS_TimeDelay(void){
 
+//I checked like two instances of timedelay and they all used OPT - dly 
+//so I assumed the other ones never get used
+void RTOS_BPS_TimeDelay(OS_TICK dly){
```

Comment:
There is an issue with using OS_TICK as an input parameter to this wrapper function if the idea is to allow this function to be used across multiple RTOS's...